ewms_pilot/pilot.py

```diff
@@ -41,15 +41,23 @@ class FileType(enum.Enum):
     JSON = ".json"
 
 
+def get_last_line(fpath: Path) -> str:
+    """Get the last line of the file."""
+    with fpath.open() as f:
+        try:
+            return f.readlines()[-1].rstrip()
```

Comment:
Do you know how big these files could be?  `readlines()` reads the entire file into memory, whereas we could iterate over the lines and only have one in memory at once.

Reply:
Arbitrarily large. I'll update this